CONTRIBUTING.md

```diff
@@ -157,6 +158,60 @@ Include details about your configuration and environment:
     * Problem can be reliably reproduced, doesn't happen randomly: [Yes/No]
     * Problem happens with all files and projects, not only some files or projects: [Yes/No]
 
+### Suggesting Features
```

Comment:
> Should we maybe call these "enhancements" to be more consistent with our labeling scheme?
 
I've debated this with myself when I was writing this section, and couldn't come up with any good arguments to use one word over the other. I like the word "feature" a bit more than "enhancement" because I think it's more familiar to most users (I never use "enhancement" when I talk to people, for example). Still, that's a good point about the label name -- I've switched to using "enhancement" instead of "feature" in 3585950. If it becomes an issue of any kind -- we can continue the quest for a better words then. :book: :mag: 

Thanks for pointing this out, @mnquintana :bow: